Utility.cpp

```diff
@@ -22,7 +27,7 @@ std::vector<std::unique_ptr<Item>> makeDefensiveItems(int num)
 {
     std::vector<std::unique_ptr<Item>> items;
     
-    while( num-- >= 0 )
+    while( --num >= 0 )
```

Comment:
Why did you change this?  
- [ ] revise

Reply:
fixed number of helpful and defense items for each character